lib/gds_api/worldwide.rb

```diff
@@ -2,11 +2,17 @@
 
 class GdsApi::Worldwide < GdsApi::Base
   def world_locations
-    get_list("#{base_url}/world-locations")
+    all_world_locations
   end
 
   def world_location(location_slug)
-    get_json("#{base_url}/world-locations/#{location_slug}")
+    world_location = all_world_locations.find do |location|
```

Comment:
This might not be a real source of worry, but how efficient is it to load all world locations every time someone (or something) requests a single world location? Is there some kind of implicit caching?